src/actions/index.js

```diff
@@ -117,13 +119,21 @@ export function signOut() {
   }
 }
 
-export function fetchProjects(parms) {
+export function fetchProjects() {
   return dispatch => {
     dispatch(setError(''))
-    dispatch(setIsFetching(false))
+    var callFetchProjects = tag => dispatch(fetchProjectsByTag(tag.value))
+    forEach(callFetchProjects, filter(propEq('display', true), GLOBALS.DISCIPLINES))
+  }
+}
+
+
+export function fetchProjectsByTag(tag) {
+  const parms = {id: MOBILE_PROJECTS, cards: true, tags: tag, sort: 'display_name'}
```

Comment:
What did we decide these mobile projects will be if the environment isn't set to production? I see there's a list of proj IDs, but I'm not clear what they are pointing to on staging.